base/multidimensional.jl

```diff
@@ -940,3 +940,178 @@ If `dim` is specified, returns unique regions of the array `itr` along `dim`.
         @nref $N A d->d == dim ? sort!(uniquerows) : (1:size(A, d))
     end
 end
+
+"""
+    groupslices(A, dim)
```

Comment:
see CONTRIBUTING.md, the signature line needs to be added to the rst docs, then run `make docs` to populate the docstring content